.github/workflows/release.yml

```diff
@@ -11,9 +11,10 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: lts/*
+          node-version: 18
```

Comment:
This really doesn't matter, using the latest Lts release of Node is just fine. All it's doing is publishing the NPM package, and GitHub release.